src/jit/function-builder.cc

```diff
@@ -132,6 +156,32 @@ FunctionBuilder::Result_t FunctionBuilder::CallHelper(wabt::interp::Thread* th,
   return static_cast<Result_t>(wabt::interp::Result::Ok);
 }
 
+
+FunctionBuilder::Result_t FunctionBuilder::CallIndirectHelper(wabt::interp::Thread* th, Index table_index, Index sig_index, uint8_t* current_pc) {
+  using namespace wabt::interp;
+  auto* env = th->env_;
+  Table* table = &env->tables_[table_index];
+  Index entry_index = th->Pop<uint32_t>();
```

Comment:
This line is requiring a lot of ugly hacks to be brought in. Would it be possible to pop this value as an `i32` inside the JITted code and then pass it into the helper as a parameter to avoid this?